### ISSUE TITLE
symfony recipe has hardcoded symfony_env to prod

### ISSUE THREAD

**Issue by Gamesh** [deployphp/deployer#1711]
| Q                 | A
| ----------------- | ---
| Issue Type        | Bug
| Deployer Version  | 6.3.0
| Local Machine OS  | mac OS X
| Remote Machine OS | Debian 7
| Deployer Recipes version | 6.1.3

### Description 
vendor/deployer/deployer/recipe/symfony.php:18
sets symfony_env to prod, which is not how it supposed to be. Each env has it's own config and params, so for test env symfony loads config_test and so on.
Basically you can't deploy to other environment than prod, because on test env every command has SYMFONY_ENV=prod prepended and loads prod config.

`export SYMFONY_ENV='prod'; cd /var/www/mds/releases/49 && (sudo setfacl -RL -m u:"www-data":rwX -m u:`whoami`:rwX var .yarn-cache web)`

symfony_env needs to be dynamic and depend on the server we're deploying to

**Comment by antonmedv** [comment]
Can you send a PR for this?

**Comment by Gamesh** [comment]
i can, but we should discuss, how we want to change that. 
Should we check for stage name?
if stage === 'production' -> 'prod'
if stage === 'test' then 'test'

or we could leave that for the end user to decide and just complain with a descriptive message and a good example
